api/mysagw/case/permissions.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.permissions import BasePermission
+
+from mysagw.case.models import Case
+
+
+class HasAccess(BasePermission):
```

Comment:
Naming: Maybe `HasAssignedIdentity`? "HasAccess" sounds a bit too generic for a permission class

Reply:
Fixed